libdnf5/base/goal.cpp

```diff
@@ -2725,9 +2722,9 @@ void Goal::Impl::add_group_upgrade_to_goal(
             old_set.emplace(pkg.get_name());
         }
         // set of package names that are part of the available version of the group
-        std::set<std::string> new_set{};
+        std::vector<std::string> new_set{};
```

Comment:
```suggestion
        std::vector<std::string> new_set;
```

No need for the brackets, default constructor should do :)